content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,156 @@
+    $ kubectl create namespace istio-system
+    $ kubectl create secret generic cacerts -n istio-system \
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/ca-cert.pem \
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/ca-key.pem \
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/root-cert.pem \
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/cert-chain.pem
+    {{< /text >}}
+
+1. Generate a `cluster.env` configuration file that informs the virtual machine
+   deployment which network CIDR to capture and redirect to the Kubernetes
+   cluster:
+
+    {{< text bash >}}
+    $ ISTIO_SERVICE_CIDR=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
+    $ echo -n ISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR > "${HOME}"/"${CLUSTER_NAME}"/cluster.env
+    {{< /text >}}
+
+1. Configure a select set of ports for exposure from the virtual machine. If you do not
+   apply this optional step, all outbound traffic on all ports is sent to the
+   Kubernetes cluster. You may wish to send some traffic on specific ports to
+   other destinations. This example shows enabling ports `3306` and `8080`
+   for capture by Istio virtual machine integration and transmission to Kubernetes.
+   All other ports are sent over the default gateway of the virtual machine.
+
+    {{< text bash >}}
+    $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> "${HOME}"/"${CLUSTER_NAME}"/cluster.env
+    {{< /text >}}
+
+1. Is this step needed if we always assume there is a load balancer? Try removing.
+    The ingress gateway port for Istiod is automatically programmed to 15012 throughout
+    the Istio software. Record the ISTIO_PILOT_PORT address in cluster.env.
+
+    {{< text bash >}}
+    $ echo "ISTIO_PILOT_PORT=15012" >> ${HOME}/${CLUSTER_NAME}"/cluster.env
+    {{< /text >}}
+
+
+1. Install Istio with virtual machine integration features enabled:
+
+    {{< text bash >}}
+    $ cat <<EOF> vmintegration.yaml
+    apiVersion: install.istio.io/v1alpha1
+
+    kind: IstioOperator
+    spec:
+      values:
+        global:
+          meshExpansion:
+            enabled: true
+          controlPlaneSecurityEnabled: true
+    EOF
+    $ istioctl install -f vmintegration.yaml
+    {{< /text >}}
+
+### Installing the virtual machine
+
+Run the following commands on the virtual machine you want to add to the Istio mesh:
+
+1. Copy the previously created `cluster.env` and `*.pem` files to the virtual machine.
+
+1. Install the Debian package containing the Istio virtual machine integration runtime.
+
+    {{< text bash >}}
+    $ curl -LO https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb
+    $ sudo dpkg -i istio-sidecar.deb
+    {{< /text >}}
+
+1. Add an IP address for Istiod to `/etc/hosts`. Replace `<INGRESS_GATEWAY>` with the
```

Comment:
Why are we only using the gateway here and not allowing direct access to Istiod for single network?

Reply:
That's ok, for consistency we should use this. There is no alternative right now - the cluster IP is not available on VMs, and pod IP changes. So it must be a LB - the gateway is one.

We should mention that InternalLoadBalancer gateway should be used if available 
@nmittler

Reply:
@linsun  @howardjohn this should probably also mention that if user set a DNS and TLS cert for istiod (for MC) - it can also be used for VMs.

Reply:
@howardjohn the two relevant threads:
https://github.com/istio/istio.io/pull/7248#discussion_r423283719
and https://github.com/istio/istio.io/pull/7248#discussion_r422639478

The suggestion from @smawson was to remove serviceip for now, as it doesn't work. If you have examples that work, post a gist, and I'll get em documented!

Reply:
Yeah, the master/remote configuration for multicluster is also now using ingress, so this is consistent with that.

WRT mentioning ILB, that's fine ... I don't think we're currently doing that on the MC docs.

Reply:
FWIW, @smawson 's proposal of using a headless service to discover istiod is a viable way to proceed. I can't sort that model out now as the release is in just a few days. I do intend to take a deeper look at that model though because it would be a good improvement to use just ingress for application discovery (the mechanics of which would involve modifying resolve.conf to list the gateway's IP address, capturing port 53 for DNS and sending it along to K8s. To implement this model, more R&D is needed and possibly some minor code changes in the manifests /and istioctl).

Cheers
-steve